2017/SEPT/SEPT17/FILLMTR/FILLMTR.py

```diff
@@ -0,0 +1,68 @@
+def main():
+    t = int(raw_input())
+    while t:
+        t -= 1
+        n = map(int, raw_input().split())
+        q = n[1]
+        n = n[0]
+        queries = []
+        good = []
+        ans = 1
+        for i in range(n):
+            good.append('a')
+        good.append('a')
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmplt5sx928/2017/SEPT/SEPT17/FILLMTR/FILLMTR.py
+++ b/tmp/tmplt5sx928/2017/SEPT/SEPT17/FILLMTR/FILLMTR.py
@@ -62,7 +62,7 @@
         else:
             print "no"
 
-            
+
 if __name__ == '__main__':
     main()
     
```